x-pack/plugins/infra/server/lib/alerting/log_threshold/reason_formatters.ts

```diff
@@ -23,11 +23,11 @@ export const getReasonMessageForUngroupedCountAlert = (
 ) =>
   i18n.translate('xpack.infra.logs.alerting.threshold.ungroupedCountAlertReasonDescription', {
     defaultMessage:
-      '{actualCount, plural, one {{actualCount} log entry} other {{actualCount} log entries}} in the last {duration}. Alert when ({translatedComparator} {expectedCount}).',
+      '{actualCount, plural, one {{actualCount} log entry} other {{actualCount} log entries}} in the last {duration}. Alert when {translatedComparator} {expectedCount}.',
     values: {
       actualCount,
       expectedCount,
-      translatedComparator: ComparatorToi18nMap[comparator],
+      translatedComparator: ComparatorToi18nSymbolsMap[comparator],
```

Comment:
@fkanout Once we get rid of i18n, `translatedComparator` would rather be called `comparator`. And of course `ComparatorToi18nSymbolsMap` to `ComparatorToSymbolsMap`